docs/AccessControl.md

```diff
@@ -24,6 +24,17 @@ convention is to:
 - promote readability and consistency among free-form names
 - interpret some typical data in the key-value style
 
+In short,
+1. `SRTO_STREAMID` is designed for a caller (client) to be able to identify himself, and tell what he wants.
+2. `srt_listen_callback(...)` function is used by a listener (server) to check what a caller (client) has provided in `SRTO_STREAMID` **before** the connection is established.
+For example, the listener (server) can check if it knows the user amd set the corresponding passphrase for a connection to be accepted.
+3. Even if `srt_listen_callback(...)` accepts the connection, SRT will still have one more step to check the PASSPHRASE, and reject on mismatch.
```

Comment:
This is a cool part of preventing mix-up. I appreciate you helping me understand this feature.

Reply:
This is a part where SRT starts checking passphrase. 
https://github.com/Haivision/srt/blob/master/srtcore/api.cpp#L473